Contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists.
```

Comment:
I think "issue a pull request" needs a section of its own - how should a PR look? what should it contain? how should the title\description look? linking an issue to a PR (this very PR needs that as well)